wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/Empty placeholders/EmptyConversationSearchResultsView.swift

```diff
@@ -0,0 +1,96 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+import WireDesign
+
+final class EmptyConversationSearchResultsView: UIView {
+
+    var newConversationAction: () -> Void
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private var hostingViewController: UIHostingController<EmptyView>!
+
+    init(newConversationAction: @escaping () -> Void) {
+        self.newConversationAction = newConversationAction
+
+        super.init(frame: .zero)
+
+        self.hostingViewController = UIHostingController(rootView: EmptyView(newConversationAction: { [weak self] in
+            self?.newConversationAction()
+        }))
+
+        self.addSubview(hostingViewController.view)
```

Comment:
question: Is it okay to just use the view of the view controller when keeping a reference to the view controller?
Is this documented somewhere?
I'd be afraid it would break one day.

Reply:
I consider it as any other UIViewController's view, looking at it : it's missing a didMoveToParent I'll add that

Reply:
ah right I am in a UIView... not a UIViewController